WebRoot/js/zimbraMail/share/model/ZmSettings.js

```diff
@@ -892,6 +892,8 @@ function() {
 	this.registerSetting("PUBLIC_URL",						{type:ZmSetting.T_CONFIG});
 	this.registerSetting("ADMIN_URL",						{type:ZmSetting.T_CONFIG});
 	this.registerSetting("DISABLE_SENSITIVE_ZIMLETS_IN_MIXED_MODE",		{type:ZmSetting.T_CONFIG});
+	this.registerSetting("DISABLE_MODERN_CLIENT",			{name:"zimbraModernWebClientDisabled", type:ZmSetting.T_DOMAIN, dataType:ZmSetting.D_BOOLEAN, defaultValue:false});
```

Comment:
A name of an attribute is usually `...Enabled` for boolean type. Is there any reason not to define `zimbraModernWebClientEnabled` with default value `TRUE`?

Reply:
yes we can do that, only concern with that is if somehow ldap attribute is not present then by default this logic will show modern ui option but if we change it to Enabled then by default it will hide the option

Reply:
I got it. Let's keep it as it is.

Reply:
Changed it to **zimbraModernWebClientEnabled** and made default value **TRUE**